packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -53,6 +53,15 @@ export function createKusama (t: TFunction): EndpointOption {
           Centrifuge: 'wss://fullnode.altair.centrifuge.io'
         }
       },
+      {
+        info: 'basilisk',
+        homepage: 'https://bsx.fi',
```

Comment:
```suggestion
        isUnreachable: true,
        homepage: 'https://bsx.fi',
```

Since the RPC is not (yet) available.

Reply:
still syncing :)